README.md

```diff
@@ -1,5 +1,15 @@
+[![Gluon](.github/assets/gluon_logo.svg)](https://gluonhq.com)
+
 # Gluon Scene Builder #
 
+[![Build](https://github.com/gluonhq/scenebuilder/actions/workflows/build.yml/badge.svg)](https://github.com/gluonhq/scenebuilder/actions/workflows/build.yml)
```

Comment:
This workflow is used to verify PRs. The workflow which shows the current state of master branch is the early-access workflow. We may have to change the name of the badge to "build" instead of default name.

Reply:
changed. It shows now `Early Access: Passing`, which is fine, since it shows that the EA release is green.